pkg/policyreport/builder.go

```diff
@@ -285,6 +287,13 @@ func (builder *requestBuilder) fetchAnnotationValues(policy, ns string) annotati
 	if severity, ok := ann[severityLabel]; ok {
 		av.setSeverityFromString(severity)
 	}
+	if scored, ok := ann[scoredLabel]; ok {
+		if scored == "true" {
+			av.scored = true
+		} else {
+			av.scored = false
+		}
```

Comment:
Can be improved to `av.scored=scored`.

Reply:
`scored` is a string and `av.scored` is a boolean. Plus to be on the safe side for backward compatibility we want that if the value of the annotation is anything apart from "false" explicitly we should revert back to the original behavior of setting scored to true.